SandroSal/ProjectNapoli
# #4 - Add all models and ORM configuration for MySQL Employees DB

## Completes #4 
### Changes Made
- Setup `sequelize` and connected AWS MySQL server 
- Created the six models from the `Employees` database's original schema
- Refactored `server.js` for our use case
- Added `RuntimeVars.js` to manage config/environmental variables
- Created a test route to ensure that the developer is connected to the dB

### Tests
1. Go to the root folder and **`npm i`**
2. Create a **`build`** folder and a subfolder inside called **`manifest`** and place **`manifest.yaml`** in there
3. In the root folder, use the command **`nodemon server.js`** or **`npm run dev`**
4. In your browser, go to [**`localhost:172/api`**](http://localhost:172/api) to check if the server is running. You should see:
```
{"success":true,"message":"test"}
```
5. Now try [**`localhost:172/api/depts`**](http://localhost:172/api/depts) to check if the database is connected. You should see:
```
[{"dept_no":"d009","dept_name":"Customer Service"},
{"dept_no":"d005","dept_name":"Development"},
{"dept_no":"d002","dept_name":"Finance"},
{"dept_no":"d003","dept_name":"Human Resources"},
{"dept_no":"d001","dept_name":"Marketing"},
{"dept_no":"d004","dept_name":"Production"},
{"dept_no":"d006","dept_name":"Quality Management"},
{"dept_no":"d008","dept_name":"Research"},
{"dept_no":"d007","dept_name":"Sales"}]
```
(all the Departments in our Employees database)
